### PR TITLE
Added Ipv6 Next Header as qualifier

### PR DESCRIPTION
**Pull request by AshokDaparthi**

### CODE REVIEW

File: inc/saiacl.h [hunk 1]
```diff
@@ -927,9 +927,18 @@ typedef enum _sai_acl_table_attr_t
     SAI_ACL_TABLE_ATTR_FIELD_ACL_RANGE_TYPE,
 
     /**
+     * @brief IPv6 Next Header
+     *
+     * @type bool
+     * @flags CREATE_ONLY
+     * @default false
+     */
+    SAI_ACL_TABLE_ATTR_FIELD_IPv6_NEXT_HEADER,
```
Comment (kcudnik) [inline inc/saiacl.h]: don't use small letters in enum members

**Comment by lguohan** [comment]
in case the packet has many ipv6 next header fields, which one?

File: inc/saiacl.h [hunk 2]
```diff
@@ -927,9 +927,18 @@ typedef enum _sai_acl_table_attr_t
     SAI_ACL_TABLE_ATTR_FIELD_ACL_RANGE_TYPE,
 
     /**
+     * @brief IPv6 Next Header
```
Comment (lguohan) [inline inc/saiacl.h]: clarify on this "the next header in IPV6 header not the next header in the extension headers."

**Comment by AshokDaparthi** [comment]
@guohan - Please check now

**Pull request merged**